pymeasure/instruments/keithley/keithley2260B.py

```diff
@@ -0,0 +1,74 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Keithley2260B(Instrument):
+    """ Represents the Keithley 2260B Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the device only accepts
+    connections at port 2268, which cannot be configured otherwise.
+        example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
```

Comment:
Hm, there is a weird jump in indentation here?

Reply:
fixed